### ISSUE TITLE
dockers build doesn't work with demo.sql for BZFE v2.4

### ISSUE THREAD

**Issue by franck-boullier** [unee-t/bugzilla-customisation#1]
We need to fix the error
---
`bugzilla_1  | Not a reference at Bugzilla/DB/Schema.pm line 2916.`.
---
When we run the BZ installation script
This tracks our progresses to do this.

**Comment by franck-boullier** [comment]
@kaihendry I've been trying to fix this for the past hours but I'm hitting a wall here...
Can you create a branch that ONLY builds a clean BZ database and do not import anything other than what BZ deems canonical at first install?
I will use this to re-create the DB customizations directly in the backend: this will be MUCH faster than banging my head on this for another day

**Comment by franck-boullier** [comment]
check if push to slack is working

**Comment by kaihendry** [comment]
If you want to start from scratch you would do something like:

1. `make down` - make sure we shut down
2. `make clean` - cleanup any mariadb mount
3. `rm sql/*` - to remove the prime https://github.com/unee-t/bugzilla-customisation/tree/master/sql

**Comment by franck-boullier** [comment]
Thanks!
I managed to create a clean, vanilla BZ 5.0.3 installation...
Re-creating the customization now...

**Comment by franck-boullier** [comment]
I have fixed the BZ schema issue: the docker build doesn't fail completely anymore.
That's the good news.
The bad new is that we're still not there as the data in the demo.sql file that I updated are not all updated in the BZ database.

To me this looks completely random: I don't know why dockers only pick some data but not all all them but I'm sure I'm missing something here.

@kaihendry let's try to nail this one later today: I want this sorted out asap and I fell we're very close...
